lib/travis/logs/app.rb

```diff
@@ -65,10 +65,10 @@ def initialize(existence = nil, pusher = nil, database = nil)
 
         job_id = Integer(params[:job_id])
 
-        log = database.log_for_job_id(job_id) || database.create_log(job_id)
+        log_id = database.log_for_job_id(job_id) || database.create_log(job_id)
```

Comment:
Looking more closely at this, I think `log_for_job_id` returns `{ id: some_num, … }` while `create_log` returns the ID only